ui/admin/app/templates/scopes/scope/credential-stores/credential-store/worker-filter.hbs

```diff
@@ -0,0 +1,46 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: BUSL-1.1
+}}
+
+{{page-title (t 'resources.credential-store.titles.worker-filter')}}
+
+<Rose::Layout::Page as |page|>
+  <page.breadcrumbs>
+    <Breadcrumbs::Container />
+  </page.breadcrumbs>
+
+  <page.header>
+    <CredentialStores::CredentialStore::Header @model={{@model}} />
+  </page.header>
+
+  <page.navigation>
+    <CredentialStores::CredentialStore::Nav @model={{@model}} />
+  </page.navigation>
+
+  <page.actions>
+    <CredentialStores::CredentialStore::Actions
+      @model={{@model}}
+      @delete={{fn this.credentialStores.delete @model}}
+    />
+  </page.actions>
+
+  <page.body class='vault-worker-filter-empty'>
```

Comment:
instead of adding a custom style, it looks like we usually wrap our app state in `<Rose::Layout::Centered>` component.

Reply:
@cameronperera suggested I remove the component and add custom styling. See [this comment](https://github.com/hashicorp/boundary-ui/pull/2576#discussion_r1848758883). Lmk what you two decide. cc @lisbet-alvarez

Reply:
Go with what @lisbet-alvarez suggested. She has been in that code recently so I trust her suggestion.

Reply:
Reverted! 👍